lib/mix/tasks/hex.build.ex

```diff
@@ -80,7 +80,7 @@ defmodule Mix.Tasks.Hex.Build do
   @max_description_length 300
   @default_repo "hexpm"
   @metadata_config "hex_metadata.config"
-
+  @spdx_licenses Code.eval_file("./src/spdx_list.exs") |> elem(0)
```

Comment:
remember to use `@external_resource` when depending on files for compilation:

```suggestion
  spdx_path = Path.join("src", "spdx_list.exs")
  @external_resource spdx_path
  @spdx_licenses Code.eval_file(spdx_path) |> elem(0)
```